cpp/cmake/treelite_protobuf.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Missing copyright header, and also add a comment explaining why this is needed along a link to any github issues if appropriate